components/search_engines/brave_prepopulated_engines.cc

```diff
@@ -131,7 +131,7 @@ const PrepopulatedEngine brave_ecosia =
 #else
                        "e8eb07a6"
 #endif
-                       "&q={searchTerms}",
+                       "&q={searchTerms}&addon=brave",
```

Comment:
Just curious how this change affetcs ecosia page?

Reply:
@bsclifton Ah,, ecosia page can read this string?

Reply:
Good question; I'm not exactly sure - I didn't notice anything special in local storage or cookies. Change is coming at request of the team.